### PR TITLE
[SVP-8075] Fix: Smaller font + smaller padding on buttons, selection lists, rows in tables etc.

### PR DESCRIPTION
**Pull request by tlipczynski**

### CODE REVIEW

File: src/theme/table.scss [hunk 1]
```diff
@@ -21,6 +23,18 @@ table .p-row-tooltip [data-pr-tooltip] {
...
+  &:first-letter {
+    text-transform: uppercase;
+  }
+}
+
+.column-cut-text {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  text-overflow: ellipsis;
```
Comment (kstepien-storware) [inline src/theme/table.scss:38]: You can extend this by using scss extends

for example:
```scss
%text-ellipsis {
  text-overflow: ellipsis;
  overflow: hidden;
}

.column-cut-text {
  // ...
  @extend %text-ellipsis;
}
``` 

if this is not compatible with our SCSS version, please use `@mixin` and `@include`

File: src/theme/theme.scss [hunk 2]
```diff
@@ -5,10 +5,17 @@
 @import "filesystem-table";
 @import "protection-doughnut";
 
+html {
+  font-size: 13px !important;
```
Comment (kstepien-storware) [inline src/theme/theme.scss:9]: `!important` is necessary?

File: src/pages/reporting/ChargebackForm.tsx [hunk 3]
```diff
@@ -108,65 +113,81 @@ export default () => {
...
-                                className='ml-2'
-                                checked={show}
-                                onChange={({ value }) => {
-                                    setShow(value);
-                                    dispatch(
-                                        value
-                                            ? getPropertyOptions(el, propertyOptions)
-                                            : setPropertyOptions({ ...propertyOptions, [el]: [] }),
-                                    );
+            {Object.keys(filterByFieldOptions).map((el) => {
```
Comment (kstepien-storware) [inline src/pages/reporting/ChargebackForm.tsx:145]: By the way, you can change `el` name to `key`
and if you want, move the below code to a separate function

**Review by kstepien-storware — CHANGES_REQUESTED** [review]

File: src/theme/theme.scss (hunk 2 above)
Comment (tlipczynski) [reply src/theme/theme.scss:9]: yes

File: src/theme/table.scss [hunk 4]
```diff
@@ -21,6 +23,22 @@ table .p-row-tooltip [data-pr-tooltip] {
...
-  font-size: 13px;
+.text-capitalize-from-uppercase {
+  text-transform: lowercase;
+
+  &:first-letter {
+    text-transform: uppercase;
+  }
+}
+
+%text-ellipsis{
```
Comment (kstepien-storware) [inline src/theme/table.scss:34]: create `utils.scss` file or something like that, and move this into this file

**Review by kstepien-storware — CHANGES_REQUESTED** [review]

**Pull request merged**